_re2.cc

```diff
@@ -65,14 +65,37 @@ typedef struct _MatchObject2 {
   StringPiece* groups;
 } MatchObject2;
 
+// Forward declaration of getter functions
+static PyObject* match_pos_get(MatchObject2* self);
+static PyObject* match_endpos_get(MatchObject2* self);
+static PyObject* match_re_get(MatchObject2* self);
+static PyObject* match_string_get(MatchObject2* self);
+static PyObject* regexp_groups_get(RegexpObject2* self);
+static PyObject* regexp_groupindex_get(RegexpObject2* self);
+static PyObject* regexp_pattern_get(RegexpObject2* self);
+
+static PyGetSetDef regexp_getset[] = {
+  {(char *)"groupindex",    (getter)regexp_groupindex_get,  (setter)NULL},
+  {(char *)"groups",        (getter)regexp_groups_get,      (setter)NULL},
+  {(char *)"pattern",       (getter)regexp_pattern_get,     (setter)NULL},
+  {NULL}
+};
+
+static PyGetSetDef match_getset[] = {
+  {(char *)"endpos",       (getter)match_endpos_get,   (setter)NULL},
+  {(char *)"pos",          (getter)match_pos_get,      (setter)NULL},
+  {(char *)"re",           (getter)match_re_get,       (setter)NULL},
+  {(char *)"string",       (getter)match_string_get,   (setter)NULL},
+  {NULL}
+};
+
 typedef struct _RegexpSetObject2 {
   PyObject_HEAD
   // True iff re2_set_obj has been compiled.
   bool compiled;
   RE2::Set* re2_set_obj;
 } RegexpSetObject2;
 
-
```

Comment:
Revert?

Reply:
I deleted the extra lines in order to be consistent with spacing but I'll revert all style changes (including the tab -> space change)